src/core/libraries/kernel/file_system.cpp

```diff
@@ -310,6 +310,37 @@ int PS4_SYSV_ABI posix_mkdir(const char* path, u16 mode) {
     return result;
 }
 
+int PS4_SYSV_ABI sceKernelRmdir(const char* path, u16 mode) {
+    auto* mnt = Common::Singleton<Core::FileSys::MntPoints>::Instance();
+    bool ro = false;
+
+    const std::filesystem::path dir_name = mnt->GetHostPath(path, &ro);
+
+    if (dir_name.empty()) {
+        return SCE_KERNEL_ERROR_EACCES;
+    }
+
+    if (ro) {
+        return SCE_KERNEL_ERROR_EROFS;
+    }
+
+    if (!std::filesystem::is_directory(dir_name)) {
+        return ORBIS_KERNEL_ERROR_ENOTDIR;
+    }
+
+    if (!std::filesystem::exists(dir_name)) {
+        return ORBIS_OK;
+    }
+
+    int result = std::filesystem::remove_all(dir_name);
+    const int error = errno;
+
+    if (error == 0) {
+        return ORBIS_OK;
+    }
+    return ErrnoToSceKernelError(error); // error - 0x7ffe0000;
```

Comment:
that's not neccesary since it is not posix function

Reply:
Pretty sure sceKernelRmdir is internally based on an rmdir posix function, so that might be where this came from.

Reply:
should be resolved by new commit